training/benchmarks/glm/pytorch/train/trainer_adapter.py

```diff
@@ -6,7 +6,10 @@
 from typing import Tuple
 
 import optimizers
-from apex.optimizers import FusedAdam as Adam
+try:
+    from apex.optimizers import FusedAdam as Adam
```

Comment:
The docker environment for other vendors may not have the apex library

Reply:
@upvenly plz double-check if this changes workable for other vendor